$OpenBSD: patch-lib_awful_tooltip_lua_in,v 1.1 2013/05/22 15:05:50 dcoppa Exp $

commit 14daf85fe91780e4ef49465c94e1cfb671391f0d
Author: Uli Schlachter <psychon@znc.in>
Date:   Mon May 13 19:27:19 2013 +0200

awful.tooltip: Set the bg color correctly (FS#1148)

Well, whoops. There is no .bg property on wiboxes for quite a while
already, it is now a :set_bg member function.

Signed-off-by: Uli Schlachter <psychon@znc.in>

--- lib/awful/tooltip.lua.in.orig	Mon May 20 16:04:44 2013
+++ lib/awful/tooltip.lua.in	Mon May 20 16:05:12 2013
@@ -140,7 +140,7 @@ local function set_defaults(self)
     self.wibox.border_width = beautiful.tooltip_border_width or beautiful.border_width or 1
     self.wibox.border_color = beautiful.tooltip_border_color or beautiful.border_normal or "#ffcb60"
     self.wibox.opacity = beautiful.tooltip_opacity or 1
-    self.wibox.bg = beautiful.tooltip_bg_color or beautiful.bg_focus or "#ffcb60"
+    self.wibox:set_bg(beautiful.tooltip_bg_color or beautiful.bg_focus or "#ffcb60")
     data[self].fg = beautiful.tooltip_fg_color or beautiful.fg_focus or "#000000"
     data[self].font = beautiful.tooltip_font or beautiful.font or "terminus 6"
 end
